types/setting.go

```diff
@@ -1443,6 +1446,16 @@ var (
 		Default:     "",
 	}
 
+	SettingDefinitionV2DataEngineFastReplicaRebuilding = SettingDefinition{
+		DisplayName: "V2 Data Engine Fast Replica Rebuilding",
```

Comment:
I assume this setting is temporary before v2 becomes GA, as delta replica rebuilding should be eventually enabled always. Right?

Reply:
YES! Next, I will check when we can launch a setting similar to v1 `Snapshot Data Integrity` instead. If we can do that before v1.8 official release, `V2 Data Engine Fast Replica Rebuilding` won't be delivered to users at all.